jormungandr-lib/src/interfaces/peer_stats.rs

```diff
@@ -29,12 +29,7 @@ pub struct Profile {
     pub subscriptions: Vec<Subscription>,
 }
 
-#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
-#[serde(deny_unknown_fields)]
-pub struct Info {
-    pub address: String,
-    pub id: String,
-}
+pub type Info = String;
```

Comment:
If you want to keep the meaning as "peer info" I'd leave it as a one-field struct for future proofing.